.github/workflows/test.yml

```diff
@@ -1,61 +1,46 @@
-name: Cargo NEAR Checks And Tests
+name: CI
 
 on:
   push:
     branches: [ main ]
   pull_request:
 
 jobs:
-  test:
+  tests:
     runs-on: ${{ matrix.platform }}
-    name: "${{ matrix.platform }} ${{ matrix.toolchain }}"
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest]
-        toolchain: [stable, 1.56.0]
+
     steps:
       - name: Checkout Sources
         uses: actions/checkout@v2
-      - name: "Install ${{ matrix.toolchain }} toolchain"
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: ${{ matrix.toolchain }}
```

Comment:
I wonder if it's worth it to retain these. By default, the runner ships with an up-to-date rust version. - https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#rust-tools

But we might want to keep this to ensure it always on the latest stable. WDYT?

Reply:
I think it is fine to use Github runner's native rust toolchain. Seems like they are keeping it up-to-date.